tests/test_modeling_tf_common.py

```diff
@@ -1223,8 +1240,17 @@ def test_save_load_after_resize_token_embeddings(self):
 
             # fetch the output for an input exclusively made of new members of the vocabulary
             inputs_dict = copy.deepcopy(original_inputs_dict)
-            new_vocab_input_ids = ids_tensor(inputs_dict["input_ids"].shape, new_tokens_size)
```

Comment:
Slight reworking below as previously was trying to get an item from a key that doesn't always exist (input_ids)